### DISCUSSION TITLE
Error: Proposal limit reached

### DISCUSSION THREAD

**Discussion by densmirnov** [snapshot-labs/snapshot#3414] [Q&A]
Hello! I am using Snapshot as the registry system for students completing the study course about DeFi. The student should register their wallet addresses through Snapshot, but after a couple of proposals have been processed, the beforementioned error  begins to appear

**Comment by ChaituVR** [comment]
Hello @densmirnov what's your space name?  Also, did you try demo.snapshot.org ? There you can create any number of spaces for free ;) maybe you can create multiple spaces and ask them to create a proposal in any of them. or maybe you can ask your students to create a space and create a proposal there. (This is just a random thought, not sure if it will be helpful in your case)

**Reply by densmirnov** [reply]
Yeah, that's a possible way. Thanks! The space name is [hsedefi.eth](https://snapshot.org/#/hsedefi.eth). Didn't know about lack of such a limitation in the demo version, but right now, the exam is already live

**Comment by maximillanolotr** [comment]
https://docs.snapshot.org/user-guides/proposals/create#proposals-limitations
